src/main/java/com/telesign/verify/response/VerifyResponse.java

```diff
@@ -1,141 +1,141 @@
-		public String code_expected;
-	}
-	
-	/** An object that describes the user's verification response. */
-	public static class UserResponse{
-		/** A timestamp marking the time when TeleSign received the user's verification response. */
-		public String received;
-		/** The pass code returned from the user. */
-		public String verification_code;
-		/** Indicates the user's intention, as selected from three choices. Possible values are ALLOWED, DENIED, and REPORTED_FRAUD. */
-		public String selection;
-	
-	}
-	
-	/**
-	 * Converts a <strong>VerifyResponse</strong> object to its equivalent JSON format.  
-	 * @see java.lang.Object#toString()
-	 * @return A string containing the response from the TeleSign server, in JSON format.  
-	 */
-	@Override
-	public String toString() {
-		return gson.toJson(this);
-	}
-}
+/**
+ * @version		1.0
+ * @copyright		Copyright © 2013, TeleSign Corporation.
+ * @license		http://opensource.org/licenses/mit-license.php The MIT License (MIT).
+ * @author		J. Weatherford
+ * @maintainer		Humberto Morales
+ * @repository		https://github.com/TeleSign/java_telesign
+ * @support		support@telesign.com
+ */
+package com.telesign.verify.response;
+
+import com.google.gson.Gson;
+
+/** The <strong>VerifyResponse</strong> class encapsulates all of the information returned from a call to either the <strong>Verify SMS</strong> web service, or to the <strong>Verify Call</strong>  web service. */
+public class VerifyResponse {
+	/** A String containing a <em>reference identifier</em> that uniquely identifies the Request message that initiated this Response. */
+	public String reference_id;
+	
+	/** A String containing the URI for accesses the PhoneID resource. */
+	public String resource_uri;
+
+	/** A String containing the name of the subresource that was accessed. For example, "standard". */
+	public String sub_resource;
+	
+	/** An array of {@link Error} objects. Each Error object contains information about an error condition that might have resulted from the Request. */	
+	public Error[] errors;
+
+	/** An object containing details about the request status. */
+	public Status status;
+	
+	/** An object that describes aspects of the user's phone. */
+	public Device device;
+	
+	/** An object that identifies your whitelabel app (customized/branded version of our AuthID application). */
+	public App app;
+
+	/** An object that describes the call forwarding status. */
+	public Call_forwarding call_forwarding;
+	
+	/** An object that describes the verification status. */
+	public Verify verify;
+	
+	/** An object that describes the user's verification response. */
+	public UserResponse user_response; 
+
+    private final transient Gson gson = new Gson();
+
+	
+	/** An array of property-value pairs, that contain information on error conditions that might have resulted from the Request. */
+	public static class Error

```

Comment:
For Verify we have one single response where with PhoneID we have one per endpoint. We should have a similar pattern here for sms, voice, twoway, etc.